### PR TITLE
update style border violet

### PR DESCRIPTION
**Pull request by XavierSevilla**
Se ha añdido border violeta

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,5 +1,8 @@
 body {
     background-color: aquamarine;
+    border-color: blueviolet;
```
Comment (OscarErre) [inline style.css]: Hey! The border-color sould be pink!

**Comment by XavierSevilla** [comment]
ok, thanks!

**Comment by XavierSevilla** [comment]
changes done.

File: style.css (hunk 1 above)
Comment (XavierSevilla) [reply style.css]: ok, Ta¡hanks

Comment (XavierSevilla) [reply style.css]: Changes done

File: style.css [hunk 2]
```diff
@@ -1,5 +1,8 @@
 body {
```
Comment (XavierSevilla) [inline style.css]: OK, perfect!

File: style.css [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
     background-color: aquamarine;
+    border-color: pink;
```
Comment (OscarErre) [inline style.css]: Ok!

**Pull request merged**